From 44efd5a7e36d131c40f35420a6022acfd9c7967b Mon Sep 17 00:00:00 2001
From: Mingcong Bai <jeffbai@aosc.io>
Date: Fri, 2 Oct 2026 19:36:31 +0800
Subject: [PATCH 4/4] AOSCOS: org.kde.kinfocenter.desktop: remove Documentation
 category

Lest KInfoCenter would be listed twice in the application menu (Kicker).

Signed-off-by: Mingcong Bai <jeffbai@aosc.io>
---
 org.kde.kinfocenter.desktop | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/org.kde.kinfocenter.desktop b/org.kde.kinfocenter.desktop
index fa93c004..f0feb6cd 100755
--- a/org.kde.kinfocenter.desktop
+++ b/org.kde.kinfocenter.desktop
@@ -238,4 +238,4 @@ Keywords[zh_CN]=kinfocenter, KDE 信息中心, 系统信息中心, xinxizhongxin
 Keywords[zh_TW]=kinfocenter;資訊中心;
 X-DBUS-ServiceName=org.kde.kinfocenter
 X-DBUS-StartupType=Unique
-Categories=Qt;KDE;System;Documentation;
+Categories=Qt;KDE;System;
-- 
2.56.0

